build.cake

```diff
@@ -63,35 +86,210 @@ Task("Test")
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "net452",
+					Runtime = "win-x86",
+					OutputDirectory = build.ToString() + "/runtimes/win-x86/lib/net452",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "net452",
+					Runtime = "win-x64",
+					OutputDirectory = build.ToString() + "/runtimes/win-x64/lib/net452",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+
+			// .NET 4.6.1
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "net461",
+					OutputDirectory = build.ToString() + "/lib/net461",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "net461",
+					Runtime = "win-x86",
+					OutputDirectory = build.ToString() + "/runtimes/win-x86/lib/net461",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "net461",
+					Runtime = "win-x64",
+					OutputDirectory = build.ToString() + "/runtimes/win-x64/lib/net461",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+
+			// .NET Standard 2.0
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netstandard2.0",
+					OutputDirectory = build.ToString() + "/lib/netstandard2.0",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netstandard2.0",
+					Runtime = "win-x86",
+					OutputDirectory = build.ToString() + "/runtimes/win-x86/lib/netstandard2.0",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netstandard2.0",
+					Runtime = "win-x64",
+					OutputDirectory = build.ToString() + "/runtimes/win-x64/lib/netstandard2.0",
+					ArgumentCustomization = args => args.Append("--no-restore"),
+				});
+
+
+			// .NET Core 2.1
+			DotNetCorePublish(
+				project.FullPath,
+				new DotNetCorePublishSettings()
+				{
+					Configuration = configuration,
+					Framework = "netcoreapp2.1",
```

Comment:
Just a question... should we add netcoreapp2.1 since it's an implementation of .netstandard?
I mean, if we add .netstandard, .netcoreapp will be automatically covered, right?

I found [.net standard implementations](https://github.com/dotnet/standard/blob/master/docs/versions.md) here

Reply:
Fixed. Only net452, net461 and netstandard2.0.

For the (Unit)Test apps netcoreapp2.1 is used.